spec/checkPullRequestReviewSpec.js

```diff
@@ -77,6 +79,27 @@ describe('Pull Request Review Module', () => {
       };
     });
 
+    describe('When reviewer requests changes and LGTM label' +
+     'was already added to the pull request.', ()=>{
+      beforeEach(async () => {
+        await robot.receive(reviewPayloadData);
+      });
```

Comment:
The LGTM label has not been added to the pull request in this payload. You need to add it before sending it to the robot.
See https://github.com/oppia/oppiabot/blob/ee82d34e3fb5108127bc27e719a35596b4269c33/spec/checkPullRequestLabelsSpec.js#L140-L161

Reply:
You also need to test for the case that a review is requested without the label, and assert that `github.issue.removeLabel` wasn't called.

Reply:
Assert has been added Sir.Sorry for the Late Reply.

Reply:
@jameesjohn  Sir Could you please guide me a bit. Actually function is written as well as tests along with the addtion of payload. But there is a single test failing which is
`
   it('Should Remove the LGTM Label', () => {
        expect(github.issues.removeLabel).toHaveBeenCalled();
      });
`
Could you please tell me what is the reason behind its failure, Because the feature is working completely fine. for any doubts about the working of functionality , Please find the attached video in the last of this PR.